Stardew Valley Expanded/[CP] Stardew Valley Expanded/i18n/de.json

```
@@ -3586,7 +3586,7 @@
   "Claire.MarriageDialogue.009": "Ich habe %kid1 und %kid2 ihr Bio-Apfeljoghurt gefüttert. Sie essen so viel!$1",
   "Claire.MarriageDialogue.010": "Unsere Kinder sind so wunderbar... es ist unsere Verantwortung, sie auf die richtige Art und Weise zu erziehen.$1",
   "Claire.MarriageDialogue.011": "Wir haben Glück, dass %kid1 und %kid2 gesund sind.$1",
-  "Claire.MarriageDialogue.012": "%Das Kind 1 hat so früh angefangen zu laufen... vielleicht wird es wie seine Mutter Tänzer?$1#$e#%Du umarmst Claire.#$e#Hehe...$7",
+  "Claire.MarriageDialogue.012": "%%kid1 hat so früh angefangen zu laufen... vielleicht wird es wie seine Mutter Tänzer?$1#$e#%Du umarmst Claire.#$e#Hehe...$7",
```

Hier muss ein '%' wieder weg
```suggestion
  "Claire.MarriageDialogue.012": "%kid1 hat so früh angefangen zu laufen... vielleicht wird es wie seine Mutter Tänzer?$1#$e#%Du umarmst Claire.#$e#Hehe...$7",
```

---

Stimmt, werde ich ändern, danke!